src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.ReportsDao;
+import core.basesyntax.dao.ReportsDaoImpl;
+import core.basesyntax.model.Record;
+import core.basesyntax.model.RecordsMapper;
+import core.basesyntax.model.RecordsMapperImpl;
+import core.basesyntax.model.RecordsValidator;
+import core.basesyntax.model.RecordsValidatorImpl;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.FruitShopServiceImpl;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.OperationStrategyImpl;
+import core.basesyntax.service.operations.BalanceOperationHandler;
+import core.basesyntax.service.operations.OperationHandler;
+import core.basesyntax.service.operations.PurchaseOperationHandler;
+import core.basesyntax.service.operations.ReturnOperationHandler;
+import core.basesyntax.service.operations.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILENAME = "./files/input.csv";
+    private static final String REPORT_FILENAME = "./files/report.csv";
+    private static final Map<Record.OperationType, OperationHandler>
+            OPERATION_HANDLER_MAP = new HashMap<>();
```

Comment:
```suggestion
            operationHandlerMap = new HashMap<>();
```
бзв якщо клас не deeply immutable, то для нього використовується звичний неймінг навіть якщо це static final 
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Спасибо, обращу внимание